zerver/lib/integrations.py

```diff
@@ -94,6 +92,23 @@ def add_doc_context(self, context):
         # type: (Dict[Any, Any]) -> None
         self.doc_context = context
 
+    def get_logo_url(self):
+        # type: () -> Optional[str]
+        logo_file_path_svg = str(pathlib.PurePath(
+            settings.STATIC_ROOT,
+            *self.DEFAULT_LOGO_STATIC_PATH_SVG.format(name=self.name).split('/')[1:]
+        ))
+        logo_file_path_png = str(pathlib.PurePath(
+            settings.STATIC_ROOT,
+            *self.DEFAULT_LOGO_STATIC_PATH_PNG.format(name=self.name).split('/')[1:]
+        ))
+        if os.path.isfile(logo_file_path_svg):
+            return self.DEFAULT_LOGO_STATIC_PATH_SVG.format(name=self.name)
+        elif os.path.isfile(logo_file_path_png):
+            return self.DEFAULT_LOGO_STATIC_PATH_PNG.format(name=self.name)
+
+        return None
```

Comment:
Actually, thinking about this a bit more -- I think we might want to add some sort of `assert logo is not None` sort of thing somewhere...